packages/redaction/README.md

```diff
@@ -23,3 +25,151 @@ const middleware = [
   }),
 ];
 ```
+
+## Utilties for redacting javascript objects
+
+To make it easier to redact large complex javascript types we provide a number of helpers.
+
+### asterixOut
+
+This ensures the redacted text is the same length and has the same line breaks as the original text,
+thereby allowing you to test the same layout cases.
+
+```ts
+import { asterixOut } from "@alwaysmeticulous/redaction";
+
+const redacted = asterixOut("some sensitive text"); // returns "**** ********* ****"
+```
+
+### redactString
+
+Redact string intelligently redacts the string by checking for common data formats. This reduces the
+risk of your app error'ing during the replay of Meticulous tests due to it failing to operate on the
+redacted data (e.g. "*******" is not a valid URL):
+
+```ts
+import { redactString } from "@alwaysmeticulous/redaction";
+
+const redacted = redactString("some sensitive text"); // returns "**** ********* ****"
+const redacted2 = redactString("test@example.com"); // returns "____@_______.com"
```

Comment:
Underscores are not a valid character in domain names so this could break some apps that are checking that the email is valid with a strict regex: https://domainname.shop/faq?id=7&currency=GBP&lang=en

We should probably use a different character?